src/renderer/features/governance/aggregates/list.ts

```diff
@@ -64,24 +65,13 @@ const $referendums = combine(
 
 sample({
   clock: networkSelectorModel.$governanceChainApi,
-  source: {
-    chain: networkSelectorModel.$governanceChain,
-  },
-  filter: ({ chain }, api) => !!api && !!chain,
-  target: referendumModel.events.stopUpdateReferendums,
-});
-
-sample({
-  clock: networkSelectorModel.$governanceChainApi,
-  source: {
-    chain: networkSelectorModel.$governanceChain,
-  },
-  filter: ({ chain }, api) => !!api && !!chain,
-  fn: ({ chain }, api) => ({ api: api!, chain: chain! }),
-  target: referendumModel.events.updateReferendums,
+  source: networkSelectorModel.$governanceChain,
+  filter: (chain, api) => nonNullable(api) && nonNullable(chain),
+  fn: (chain, api) => ({ chain: chain!, api: api! }),
+  target: referendumModel.events.subscribeReferendums,
 });
```

Comment:
In which place now will we update/resubscribe the referendum list when we change the network?